packages/block-library/src/latest-posts/edit.js

```diff
@@ -116,21 +114,20 @@ class LatestPostsEdit extends Component {
 					/>
 					{ displayPostContent &&
 					<RadioControl
-        		label="Show:"
-        		selected={ displayFullPost }
-        		options={ [
-								{ label: 'Excerpt', value: 'e' },
-            		{ label: 'Full Post', value: 'f' },
-        		] }
-        		onChange={ ( value ) => this.updateAttribute( 'displayFullPost', value ) }
-    			/>
+						label="Show:"
+						selected={ displayPostContentRadio }
+						options={ [
+							{ label: 'Excerpt', value: 'e' },
+							{ label: 'Full Post', value: 'f' },
+						] }
+						onChange={ ( value ) => this.updateAttribute( 'displayPostContentRadio', value ) }
```

Comment:
**Minor**: currently you are creating a new anonymous function every time the render function runs.

Instead of creating a new anonymous function here could you [partially apply](https://medium.com/javascript-scene/curry-or-partial-application-8150044c78b8) the `displayPostContentRadio` argument, thereby allowing your handling function to be awaiting only 1 argument (ie: `value`). You could then do:

`onChange={ this.updateDisplayPostContentAttr }`

...thereby avoiding the anonymous function which should help performance wise.

Reply:
I will look into this!

Reply:
I am going to scrap the updateAttribute function and make all the changes in place as @draganescu suggested below